openid-4-verifiable-presentations-1_0.md

```diff
@@ -1900,88 +1900,137 @@ these constraints. The Wallet is not controlled by the Verifier and the Verifier
 MUST perform its own security checks on the returned Credentials and
 Presentations.
 
-## DCQL Value Matching {#dcql-value-matching}
+# Privacy Considerations {#privacy-considerations}
+
+Many privacy considerations are specific to the Credential format and associated proof type used in a particular Presentation.
+
+This section focuses on privacy considerations specific to the presentation protocol while also addressing cross-cutting concerns related to credential formats, Wallet behavior, and Verifier practices.
+
+Wallet providers and Verifiers need to take into account privacy considerations in this section to mitigate the risks of
+data leakage, user tracking, and other privacy harms. These considerations inform the development of privacy risk analysis.
+
+## User Consent {#user_consent_and_choice}
+
+Wallets SHOULD obtain explicit, informed consent from the End-User before releasing any Verifiable Credential or Presentation to a Verifier.
```

Comment:
```suggestion
Wallets SHOULD obtain explicit, informed consent from the End-User before releasing any response to a Verifier, specifically those which contain a Verifiable Credential or Presentation.
```

I think we should be broader here as certain errors can leak possible information about the user back to the Verifier.

Reply:
Right when i said response in this sentence I was meaning the OpenID4VP response not resolution of the request object if thats what you meant?

Perhaps this is clearer

```suggestion
Wallets SHOULD obtain explicit, informed consent from the End-User before releasing a Verifiable Credential or Presentation to a Verifier.
```

As the language implies more clearly there is really only one response the wallet sends in the protocol.